test/puppetlabs/trapperkeeper/services/status/status_proxy_service_test.clj

```diff
@@ -20,9 +20,9 @@
    :ssl-ca-cert (str dev-resources "/ssl/certs/ca.pem")})
 
 (def ssl-status-service-config
-  {:webserver (merge common-ssl-config
-                {:ssl-host "0.0.0.0"
-                 :ssl-port 9001})
+  {:webserver          (merge common-ssl-config
```

Comment:
I don't think this indentation conforms to our indentation standards, I don't think we should be aligning the values in a map with each other.